src/aiperf/plot/dashboard/components.py

```
@@ -0,0 +1,532 @@
+                id={"type": "section-content", "id": section_id},
+                style={"display": "block" if initially_open else "none"},
+            ),
+        ]
+    )
+
+
+def create_run_selector_checklist(
+    run_options: list[dict],
+    selected_indices: list[int],
+    theme: PlotTheme,
+    runs: list = None,
```

Ruff (0.14.7)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)